README.md

```diff
@@ -5,65 +5,162 @@ The goal of this project is to ease the process of bootstrapping an Aeternity hy
+4. Managing validator and pinner account setup
+
+Each component is configured through YAML files and automated scripts to minimize manual intervention while maintaining flexibility.
+
+## Usage
+### Quick Start
+1. Clone and install the starter kit:
+
+   ```shell
+   git clone https://github.com/aeternity/hyperchain-starter-kit
+   cd hyperchain-starter-kit
+   npm install
+   npm run dev
+   ```
+
+2. Initialize your Hyperchain configuration:
+
+   ```shell
+   npm run dev init your_chain_name
+   ```
+
+3. Set up contracts:
+
+   ```shell
+   npm run dev retrieve-contracts your_chain_name
+   ```
+
+4. Generate the economy:
+
+   ```shell
+   npm run dev gen-economy your_chain_name
+   ```
+
+5. Generate node configuration:
+
+   ```shell
+   npm run dev gen-node-conf your_chain_name
+   ```
+
+See the [Aeternity Node Documentation](https://github.com/aeternity/aeternity/tree/master/docs) for detailed setup instructions.
+
+**Important Notes:**
+- Back up all generated keys and configuration files
+- Fund all pinner accounts on the parent chain before starting your node
+- When using a public parent chain, the start height is set to current block + 10
+
+## Example Configuration
+The tool generates a default configuration in `init.yaml`:
+
+```yaml
+childBlockTime: 3000
+childEpochLength: 600
+enablePinning: true
+networkId: 'hc_test'
+parentChain:
+  epochLength: 10
+  networkId: 'ae_uat'
+  nodeURL: 'https://testnet.aeternity.io'
+  type: 'AE2AE'
+pinningReward: 1000000000000000000000
+validators:
+  count: 3
+  validatorMinStake: 1000000000000000000000000
+```
+
+This configuration can be customized based on your requirements. See the [Hyperchains documentation](https://github.com/aeternity/aeternity/tree/master/docs) for more detailed parameter explanations.
 
 ## Non-goals
-- We do not aim to teach you all the concepts behind hyperchains, aeternity or blockhains in general. Please read the
-relevant documentation if you want to learn more about those.
+### Project Scope
+While this starter kit aims to simplify Hyperchain deployment, it is not:
+- A comprehensive tutorial on blockchain concepts
+- A complete guide to Aeternity node operation
+- A production-ready deployment solution
+- A replacement for understanding basic blockchain operations
+
+For learning these concepts, we recommend:
+- [Aeternity Documentation](https://github.com/aeternity/aeternity/tree/master/docs) for blockchain fundamentals
+- [Hyperchains Whitepaper](https://github.com/aeternity/hyperchains-whitepaper) for deep technical understanding
+- [Aeternity Forum](https://forum.aeternity.com/) for community learning resources
 
 ## Prerequisites/assumptions
 - You have a somewhat solid understanding of how blockchains work and [æternity](https://github.com/aeternity/aeternity) in particular.
-- You are comfortable with the command line, you are a semi-competent unix administrator.
+- A running Aeternity node (v6.7.0 or later)
```

Comment:
This tool doesn't actually needs a node, it outputs the artefacts that the node needs, so more or less it's inverse dependancy